app/backend/middleware/graphql/authenticationPgSettings.js

```diff
@@ -0,0 +1,48 @@
+import { isAuthenticated } from "@bcgov-cas/sso-express";
+
+const authenticationPgSettings = (req) => {
+
+  const claimsSettings = {};
+  if (!isAuthenticated(req))
+    return {
+      ...claimSettings,
+    };
+
+  const claims = req.claims;
+
+  claims.user_groups = groups.join(",");
+  claims.priority_group = priorityGroup;
```

Comment:
we don't need groups since we're in a shared realm and can't set these but we do need a role always